FEWD/Week07/lesson13/classwork/responsive_articles/css/styles.css

```diff
@@ -0,0 +1,206 @@
+/* Box Model Hack */
+* {
+ box-sizing: border-box;
+}
+
+/* Clearfix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  background-color: #F6F0E4;
+}
+
+/*HEADER styling*/
+header {
+  background-color: #7BAFA4;
+}
+
+ul {
+  padding: 0px 20px 20px;
+  margin: 0;
+  /*overflow: hidden;
+  height: 0;
+  transition: all 5s ease-in-out;*/
+}
+
+li {
+  padding: 7.5px 0;
+  border-bottom: 2px solid #F6F0E4;
+}
+
+li:first-child {
+  border-top: 2px solid #F6F0E4;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+  font-weight: bold;
+  padding: 0;
+}
+
+li:hover {
+  background-color: #488A7C;
+  cursor: pointer;
+}
+
+.hamburger {
+  background: none;
+  color: white;
+  border: none;
+  font-size:2.3em;
+  cursor: pointer;
+  text-decoration: none;
+  outline:none;
+  padding: 0;
+  margin-left: 90%;
+  }
+/*
+.visible {
+  overflow: visible;
+  padding-bottom: 20px;
+  height: 160px;
+  }*/
+
+/* Basic/Mobile Article Styling: */
+
+article {
+  padding: 20px;
+}
+
+h2 {
+  font-family: 'PT Serif', serif;
+  font-size: 2em;
+  margin: 20px 0;
+}
+
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+/* Responsive Article Styling: */
+
+@media (min-width: 480px) { /* mobile-landscape */
+  /* limit <main> to 440px and do necessary styling to match screenshot
+  Use classes and elements present in HTML as clues */
+  body {
+    font-size: 12.5px;
+  }
+
+  main {
+    width: 440px;
+    margin: 0 auto;
+  }
+
+  img {
+    display: inline-block;
+    width: 50%;
+  }
+
+  .left-aligned, .right-aligned {
+    display: inline-block;
+    width: 50%;
+    float: left;
```

Comment:
Best to remove this `float: left` here and apply it to `left-aligned` separately, so that you're not relying to the fact that the definition for `.right-aligned` just happens to be below and is therefore overriding this